manual/src/cmds/runtime-tracing.etex

```diff
@@ -314,4 +322,146 @@ Percentile 	 Latency (ms)
 99.9990 	 9.17
 99.9999 	 9.17
 100.0000 	 9.17
-\end{verbatim}
\ No newline at end of file
+\end{verbatim}
+
+\section{s:runtime-tracing-custom-events}{Custom events}
+
+\subsection{s:runtime-tracing-custom-events-overview}{Overview}
+
+The runtime events system supports defining custom events. Identified by a
+globally unique name, they are emitted with payloads of built-in (\texttt{unit},
+\texttt{int}, \texttt{span}) and user-defined types. To understand the
+manipulation of custom events, it is useful to know how they are transported and
+stored: their representation consists of a name string (in fact, an index into
+an array of all custom names) and an arbitrary array of bytes. Custom event
```

Comment:
"array of bytes": does this means a `byte array`? If it is just `bytes`, then "an arbitrary byte sequence" would feel more natural to me.

Reply:
Changed in 4538ac4728.

Reply:
I suspect that there are other occurrences of "array of bytes" in the documentation that probably need to be fixed as well.

Reply:
True. 155849b